src/App/App.js

```diff
@@ -156,7 +156,21 @@ function FullApp({ pendingTxns, setPendingTxns }) {
   );
   const [redirectModalVisible, setRedirectModalVisible] = useState(false);
   const [shouldHideRedirectModal, setShouldHideRedirectModal] = useState(false);
-  const [redirectPopupTimestamp, setRedirectPopupTimestamp] = useLocalStorage(REDIRECT_POPUP_TIMESTAMP_KEY);
+  const [redirectPopupTimestamp, setRedirectPopupTimestamp] = useLocalStorage(REDIRECT_POPUP_TIMESTAMP_KEY, undefined, {
+    deserializer: (val) => {
+      if (!val) {
+        return undefined;
+      }
+      const num = parseInt(val);
+
+      if (Number.isNaN(num)) {
+        return undefined;
+      }
+
+      return num;
+    },
+    serializer: (val) => (val ? val.toString() : undefined),
+  });
```

Comment:
but what happened? is there some NaN got saved?)

Reply:
initially there would be no value, but it sstill would be treated as number, i think it worked only by luck and js type coersion